client/components/product-list.jsx

```diff
@@ -0,0 +1,42 @@
+
+import React from 'react';
+import ProductListItem from './product-list-item';
+
+export default class ProductList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      products: [],
+      message: null
+    };
+
+    this.getProducts = this.getProducts.bind(this);
+  }
+
+  getProducts() {
+
+    fetch('/api/products')
+      .then(res => res.json())
+      .then(data => this.setState({ products: data }))
+      .catch(err => this.setState({ message: err.message }));
+  }
+
+  componentDidMount() {
+    this.getProducts();
+  }
+
+  render() {
+
+    return (
+      <div>
+        <div className="container">
+          <div className="row">
+            <ProductListItem products={this.state.products}/>

```

Comment:
Here is where you would map through this.state.products and then for each one you would use a <ProductListItem> with a single products data passed in as a prop. :)